package.json

```diff
@@ -2,126 +2,228 @@
-    "build:proto-types:circuit": "pbts -o src/circuit/protocol/index.d.ts src/circuit/protocol/index.js",
-    "build:proto-types:fetch": "pbts -o src/fetch/proto.d.ts src/fetch/proto.js",
-    "build:proto-types:identify": "pbts -o src/identify/message.d.ts src/identify/message.js",
-    "build:proto-types:plaintext": "pbts -o src/insecure/proto.d.ts src/insecure/proto.js",
-    "build:proto-types:peer": "pbts -o src/peer-store/pb/peer.d.ts src/peer-store/pb/peer.js",
-    "build:proto-types:peer-record": "pbts -o src/record/peer-record/peer-record.d.ts src/record/peer-record/peer-record.js",
-    "build:proto-types:envelope": "pbts -o src/record/envelope/envelope.d.ts src/record/envelope/envelope.js",
-    "test": "aegir test",
-    "test:ts": "aegir build --no-bundle && npm run test --prefix test/ts-use",
-    "test:node": "aegir test -t node -f \"./test/**/*.{node,spec}.js\"",
-    "test:browser": "aegir test -t browser",
-    "test:examples": "cd examples && npm run test:all",
-    "test:interop": "LIBP2P_JS=$PWD npx aegir test -t node -f ./node_modules/libp2p-interop/test/*",
-    "prepare": "npm run build",
-    "coverage": "nyc --reporter=text --reporter=lcov npm run test:node"
-  },
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/libp2p/js-libp2p#readme",
   "repository": {
     "type": "git",
-    "url": "https://github.com/libp2p/js-libp2p.git"
+    "url": "git+https://github.com/libp2p/js-libp2p.git"
+  },
+  "bugs": {
+    "url": "https://github.com/libp2p/js-libp2p/issues"
   },
   "keywords": [
+    "IPFS",
     "libp2p",
     "network",
     "p2p",
     "peer",
-    "peer-to-peer",
-    "IPFS"
+    "peer-to-peer"
   ],
-  "bugs": {
-    "url": "https://github.com/libp2p/js-libp2p/issues"
-  },
-  "homepage": "https://libp2p.io",
-  "license": "MIT",
   "engines": {
-    "node": ">=15.0.0"
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
   },
-  "browser": {
-    "nat-api": false
+  "type": "module",
+  "types": "./dist/src/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
+      ],
+      "src/*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
+      ]
+    }
+  },
+  "files": [
+    "src",
+    "dist/src",
+    "!dist/test",
+    "!**/*.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/src/index.js"
+    },
+    "./insecure": {
+      "import": "./dist/src/insecure/index.js"
+    },
+    "./pnet": {
+      "import": "./dist/src/pnet/index.js"
+    },
+    "./pnet/generate": {
+      "import": "./dist/src/pnet/key-generator.js"
+    }
   },
   "eslintConfig": {
     "extends": "ipfs",
+    "parserOptions": {
+      "sourceType": "module"
+    },
     "ignorePatterns": [
       "!.aegir.js",
       "test/ts-use",
       "*.d.ts"
     ]
   },
+  "release": {
```

Comment:
Seems like libp2p repo doesn't use semantic release so this is useless

Reply:
Good spot - it was added by [check-aegir-project](https://www.npmjs.com/package/check-aegir-project), I've updated it to only do that if there's an npm script called `"release"` with the contents `"semantic-release"`